Assets/Scripts/BlockFaceBehaviour.cs

```diff
@@ -72,7 +72,7 @@ public bool FireRaycastFromFace(float skinToLengthRatio, LayerMask ignores)
         foreach (Vector3 skinVertex in skinVertices)
         {
             Debug.DrawRay(skinVertex, _normal * 0.5f, Color.red, 1f);
-            if (Physics.Raycast(skinVertex, _normal, _faceLength, ignores))
+            if (Physics.Raycast(skinVertex, _normal, _faceLength, collidable))

```

Comment:
same here

Reply:
changes made as requested. Can squash if you want